package.json

```diff
@@ -39,8 +45,9 @@
     "eslint": "^8.33.0",
     "eslint-plugin-prettier": "^4.2.1",
     "jest": "^29.3.1",
-    "prettier": "^2.8.3",
-    "supertest": "^6.3.1",
+    "prettier": "^2.6.2",
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏accidentally downgraded prettier.